app/src/main/java/com/android/unio/ui/home/Home.kt

```diff
@@ -197,7 +175,7 @@ fun HomeScreen(
             LazyColumn(
                 contentPadding = PaddingValues(vertical = 8.dp),
                 modifier = Modifier.fillMaxSize().padding(horizontal = 32.dp)) {
-                  items(events) { event -> EventCard(event = event, viewModel = eventViewModel) }
+                  items(events) { event -> EventCard(event = event, userViewModel = userViewModel) }
```

Comment:
Same thing as before, why use a user view model to fetch events? This seems odd.